src/improvements/tasks/sep/016-U16-opcm-upgrade-v400-base/README.md

```diff
@@ -21,16 +21,77 @@ which reads the inputs from the [`config.toml`](./config.toml) file.
 
 ## Signing and execution
 
-Follow the instructions in the [Nested Execution](../../../NESTED.md) guide for the following steps:
+Signing and execution instructions for this task are unique because of the extra layer of nesting.
+The L1 ProxyAdmin Owner (L1PAO) is setup as following for Base Sepolia:
+
+```text
+┌───────┐┌───────┐┌───────┐┌───────┐┌───────┐┌───────┐
+│Signer1││Signer2││Signer3││Signer4││Signer5││Signer6│
+└┬──────┘└┬──────┘└┬──────┘└┬──────┘└┬──────┘└┬──────┘
+┌▽────────▽┐┌──────▽────────▽┐┌──────▽────────▽┐
+│Base      ││SC              ││FND             │
+└┬─────────┘└┬───────────────┘└┬───────────────┘
+┌▽───────────▽┐                │
+BaseNested    │                │
+└┬────────────┘                │
+┌▽─────────────────────────────▽┐
+│L1PAO                          │
+└┬──────────────────────────────┘
+┌▽─────────┐
+│ProxyAdmin│
+└──────────┘
+```
+
+where:
+
+- L1PAO is `0x0fe884546476dDd290eC46318785046ef68a0BA9`
+- FND is `0x6AF0674791925f767060Dd52f7fB20984E8639d8`
+- BaseNested is `0x646132A1667ca7aD00d36616AFBA1A28116C770A`
+- Base is `0x6AF0674791925f767060Dd52f7fB20984E8639d8` (on Sepolia, the Base and FND safes are the same)
+- SC is `0x5dfEB066334B67355A15dc9b67317fD2a2e1f77f`
+- `SignerN` represent arbitrary signers for Safes
+
+### Facilitator Instructions
+
+1. Simulate this task from the `BaseNested` 2/2 by running `SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate base_nested`
+2. In the terminal logs, directly above the Domain Hash and Message Hash, this will log the Safe Transaction Hash.
+3. Run `just new task ApproveSafeHash` to create a new task, and update the config file so this Safe Transaction Hash is used.
```

Comment:
This task template does not yet exist and needs to be created

Reply:
Might be easier for task N to be the approve hash txn, task N+1 is the actual transaction